Fix grammar

IvanCherepov opened the pull request:
Fix multiple typos in the documentation.

michaelmdresser reviewed (changes requested):
Thank you so much for this PR! Documentation quality is really important to us and it is a continuing battle to keep it all up-to-date and well-written. All the help we can get is appreciated. Do you have any recommendations for linting documentation PRs?

Three things about this PR:
- Our current primary branch for this repo is actually `main`, but we haven't yet made it the default in GitHub. If its possible to re-target this PR without breaking everything, I'd appreciate it if you could do so. Otherwise, we can merge into `master` and we'll handle getting these changes into `main`.
- Looks like you've got a couple of merge conflicts, probably because we took so long to review this. Sorry about that!
- I have a few small requests for rephrasing or not doing some edits. Let me know if those work out okay for you!

IvanCherepov commented on alerts.md:
```diff
@@ -2,18 +2,18 @@
  1. [Recurring update](#type-recurring-update) - sends an email and/or Slack alert with cluster spend across all or a set of namespaces, with costs broken down by namespace
 
- 2. [Budget](#type-budget) -- sends an email and/or Slack alert when spend crosses a defined threshold
+ 2. [Budget](#type-budget) -- sends an email and/or Slack alert when spending crosses a defined threshold
```
fixed

IvanCherepov commented on multi-cloud.md:
```diff
@@ -1,5 +1,5 @@
 # Setting Up Cloud Integrations
-This document outlines how to set up cloud integration for accounts on multiple cloud providers, or multiple acounts on the same cloud provider. Multi-Cloud is an enterprise feature. This configuration can be used independent of or in addition to other cloud integration configurations provided by kubecost. Once configured Kubecost will display cloud assets for all configured account and perform r ...
+This document outlines how to set up cloud integration for accounts on multiple cloud providers, or multiple accounts on the same cloud provider. Multi-Cloud is an enterprise feature. This configuration can be used independently of or in addition to other cloud integration configurations provided by Kubecost. Once configured Kubecost will display cloud assets for all configured accounts and perfo ...
```
fixed

IvanCherepov commented on allocation-api.md:
```diff
@@ -42,15 +42,15 @@ Optional request parameters include the following:
 > NOTE: this API is actively being replaced by the [Kubecost Allocation API](https://github.com/kubecost/docs/blob/master/allocation.md). That is the recommended API for querying historical and run-rate cost allocation metrics.
 
-The aggregated cost model API retrieves data similar to the Kubecost Allocation frontend view (e.g. cost by namespace, label, deployment and more) and is available at the following endpoint:
+The aggregated cost model API retrieves data similar to the Kubecost Allocation frontend view (e.g. cost by namespace, label, deployment, and more) and is available at the following endpoint:
```
:)

IvanCherepov commented:
@michaelmdresser You are welcome! Unfortunately, I don't know good git linter for PRs. Although I think I addressed all of your comments, rebase might take a lot of effort. Do you think it'll be a good idea to close this PR and start a new one from the correct branch?

michaelmdresser reviewed (approved):
This is perfect now @IvanCherepov, thank you so much for the contribution.
